### PR TITLE
[[ Bug 14786 ]] Fix random crash when changing textsize on Android

### PR DESCRIPTION
**Pull request by livecodeian**

### CODE REVIEW

File: libgraphics/src/mblandroidtext.cpp [hunk 1]
```diff
@@ -191,9 +191,12 @@ MCHarfbuzzSkiaFace *MCHarfbuzzGetFaceForSkiaTypeface(SkTypeface *p_typeface, uin
 	if (t_success)
 	{
 		MCHarfbuzzSkiaFace *t_hb_sk_face;
-		t_hb_sk_face = (MCHarfbuzzSkiaFace *)MCGCacheTableGet(s_hb_face_cache, t_key, sizeof(t_id));
-		if (t_hb_sk_face != nil)
+		// IM-2015-10-02: [[ Bug 14786 ]] Make sure we store & dereference the *pointer* to MCHarfbuzzSkiaFace
+		//    instead of its contents.
+		t_hb_sk_face_ptr = (MCHarfbuzzSkiaFace **)MCGCacheTableGet(s_hb_face_cache, t_key, sizeof(t_id));
```
Comment (livecodepanos) [inline libgraphics/src/mblandroidtext.cpp:197]: Yeah you found it! One thing though, I think you forgot the declaration of `MCHarfbuzzSkiaFace **t_hb_sk_face_ptr;`

**Comment by livecodeian** [comment]
@livecode-vulcan try 57a54b4

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 57a54b4

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 57a54b4fef4e3c9189cb6d0eaca27e13eb3a53eb

**Comment by livecode-vulcan** [comment]
😎 test success 57a54b4fef4e3c9189cb6d0eaca27e13eb3a53eb
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/583)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/595)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/584)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/584)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/593)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/604)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/595)

**Pull request merged**